erudio-client.json

```diff
@@ -1,4 +1,4 @@
 {
-  "erudio_base_url":"https://api.dev.edtech.fagbokforlaget.no/",
-  "erudio__access_token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2MGMzMTY0ZTYyNGRjNWUyNGJjYjBhZWUiLCJyb2xlcyI6WyJzdHVkZW50Il0sImV4dGVybmFsX2lkIjoibjZhWWV1dzNZN3N3VkhSTVpvMHFIeE5Wb19FdE1YNVZ6VUJxVmJaZllwYyIsImF1ZCI6ImxvY2FsaG9zdDo4MDAwIiwiaXNzIjoiYXV0aC1wcm94eS5kZXYuZWR0ZWNoLmZhZ2Jva2ZvcmxhZ2V0Lm5vIiwicHJpbWFyeV9yb2xlIjoic3R1ZGVudCIsInByb3ZpZGVyIjoiYXp1cmVhZCIsInRlbmFudCI6ImZvcmxhZ3NodXNldCIsImlhdCI6MTY2MTQwNzgyMiwiZXhwIjoxNjYyNjE3NDIyfQ.qYMzjTYkqwH5fj4S_sC9zpS7Fnpqmif8g3dCbT7EIQc"
-}
\ No newline at end of file
+  "erudio_base_url": "https://api.dev.edtech.fagbokforlaget.no/",
+  "erudio__access_token": ""
```

Comment:
@JERILJOSE96 I don't think that's enough, you should remove the leaked token from the repo

https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository